src/sdk/pynni/nni/compression/torch/quantization/quantizers.py

```diff
@@ -339,11 +360,12 @@ def validate_config(self, model, config_list):
 
         schema.validate(config_list)
 
-    def quantize_weight(self, weight, wrapper, **kwargs):
-        out = torch.sign(weight)
+    def quantize_weight(self, wrapper, **kwargs):
+        weight = wrapper.module.weight.data
+        weight = torch.sign(weight)
         # remove zeros
-        out[out == 0] = 1
-        return out
+        weight[weight == 0] = 1
+        wrapper.module.weight.data = weight
```

Comment:
`return` is removed? returned value is never used?

Reply:
Returned value is quantized weight. There is no need to return in this version because weight is quantized in place in the function quantize_weight().